VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml

```diff
@@ -10,7 +10,7 @@ bootstrap_barrio_button: 1
 bootstrap_barrio_button_size: ''
 bootstrap_barrio_button_outline: 0
 bootstrap_barrio_navbar_toggle: navbar-toggleable-md
-bootstrap_barrio_navbar_container: 1
+bootstrap_barrio_ngit@github.com:Ahmed-Ayman/vartheme_bs4.gitavbar_container: 1
```

Comment:
Strange to add Ahmed-Ayman/vartheme_bs4 in the setting file

Reply:
Ops, sounds like a typo. I'll fix it.

Reply:
This was a typo that I've fixed later in the fix typo commit.